### PR TITLE
Feat/password recovery

### PR DESCRIPTION
**Pull request by ninadarbaidze**

### CODE REVIEW

File: components/Landing/ForgotPassword/ForgotPassword.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  Input,
+  Button,
+  useForgotPassword,
+  BackIcon,
+} from 'components';
+
+const ForgotPassword = () => {
+  const { formik, t, loginState, forgotPassState } = useForgotPassword();
+
+  const backToLoginHandler = () => {
```
Comment (GigaGiorgadze) [inline components/Landing/ForgotPassword/ForgotPassword.tsx]: handler goes to custom hoook above

File: components/Landing/ForgotPassword/index.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export { default as ForgotPassword } from './ForgotPassword';
+export * from './useForgotPassword';
```
Comment (GigaGiorgadze) [inline components/Landing/ForgotPassword/index.ts]: do u really need to export useForgotPassword? i don't think u should need it outside of forgotPassword page and if u do it should be in global hooks not here

File: components/Landing/Login/Login.tsx [hunk 3]
```diff
@@ -1,7 +1,104 @@
...
+    changePasswordRecoveryState,
+  } = useLogin();
+
+  const handlePopupState = () => {
+    changeLoginState(false);
+    changeSignUpState(true);
+  };
+
+  const handlePasswordPopupState = () => {
+    changeLoginState(false);
```
Comment (GigaGiorgadze) [inline components/Landing/Login/Login.tsx]: same here and everywhere else, functions that change state A.K.A stateful functions go to custom hooks stateless functions go to helpers file

File: components/Landing/Login/index.ts [hunk 4]
```diff
@@ -1 +1,2 @@
 export { default as Login } from './Login';
+export * from './useLogin';
```
Comment (GigaGiorgadze) [inline components/Landing/Login/index.ts]: same here, don't export

File: public/locales/en/home.json [hunk 5]
```diff
@@ -1 +1 @@
...
...":"Password","login":"Log in","logIn":"Log in","loginBtn":"Sign in","loginFooter":"Do not have an account?","loginFooter1":"Sign up","loginGoogle":"Log in with Google","loginH1":"Log in to your account","loginH2":"Welcome back! Please enter your details.","loginInp1":"Email","loginInp2":"Password","namePlaceholder":"At least 3 & max.15 lower case characters","newPassBtn":"Reset password","newPassH1":"Create new password","newPassH2":"Your new password must be different from \nprevious used passwords","passBtn":"Go to my news feed","PassErrBtn":"Go to home page","passErrH1":"Failed","passErrH2":"Sorry, something went wrong, please try again","passSuccH1":"Success!","passSuccH2":"Your Password changed successfully","passwordPlaceholder":"At least 8 & max.15 lower case characters","remember":"Rememer me","signup":"Sign Up","signupFooter":"Already have an account?","signupGoogle":"Sign up with Google","start":"Get started","startJourney":"Start your journey!","thanks":"Thank you!","thanksBtn1":"Go to my email","thanksBtn2":"Go to my news feed","thanksText1":"Please check your email and follow the instructions to\nactivate your account.","thanksText2":"Your account has been activated."}
```
Comment (GigaGiorgadze) [inline public/locales/en/home.json]: for the love of god please format this file, why its only on first line xD same for georgian

**Review by GigaGiorgadze — CHANGES_REQUESTED** [review]

File: public/locales/en/home.json (hunk 5 above)
Comment (ninadarbaidze) [reply public/locales/en/home.json]: I'm using 18 nexus app for translations and then exporting from there with specific command. Every time I change something there and export it, file looses it's format. I'll still format it though

**Review by GigaGiorgadze — APPROVED** [review]

File: public/locales/en/home.json (hunk 5 above)
Comment (GigaGiorgadze) [reply public/locales/en/home.json]: WOW, didn't know about this app, looks good, but please don't forget to format, it will be hard to read like this,

**Pull request merged**